src/main/java/com/mraof/minestuck/world/gen/feature/MSPlacedFeatures.java

```diff
@@ -144,6 +145,11 @@
 	public static final RegistryObject<PlacedFeature> SMALL_RABBIT_PLACEMENT = REGISTER.register("small_rabbit_placement", () -> placed(MSCFeatures.RABBIT_PLACEMENT,
 			worldGenModifiers(RarityFilter.onAverageOnceEvery(2), PlacementUtils.HEIGHTMAP)));
 	
+	public static final RegistryObject<PlacedFeature> CRUXITE_ORE = REGISTER.register("cruxite_ore", () -> placed(MSCFeatures.CRUXITE_ORE,
+			worldGenModifiers(CountPlacement.of(OreGeneration.cruxiteVeinsPerChunk), InSquarePlacement.spread(), HeightRangePlacement.uniform(VerticalAnchor.bottom(), VerticalAnchor.absolute(OreGeneration.cruxiteStratumMax)), BiomeFilter.biome())));
+	public static final RegistryObject<PlacedFeature> URANIUM_ORE = REGISTER.register("uranium_ore", () -> placed(MSCFeatures.URANIUM_ORE,
+			worldGenModifiers(CountPlacement.of(OreGeneration.uraniumVeinsPerChunk), InSquarePlacement.spread(), HeightRangePlacement.uniform(VerticalAnchor.bottom(), VerticalAnchor.absolute(OreGeneration.uraniumStratumMax)), BiomeFilter.biome())));
```

Comment:
Check out what that placement modifier helper function does. It already takes care of some of those modifiers.

Reply:
Let me know if I missed any other modifiers already handled by worldGenModifiers()